docs(content-releases): add frontend docs (#19391)
strapi/strapi: docs/docs/docs/01-core/content-releases/02-frontend/01-list-page.mdx

Reviewer: I would name this file `01-releases-page` since it is the name of the page in the codebase

---

> title: Releases list view

Reviewer: ```suggestion
title: Releases page
```

tags:
  - content-releases
  - tech design
---

### Overview

The releases page provides a comprehensive display of all available content releases. Content is organized into two tabs: 'pending' (releases not yet published) and 'done' (releases already published).

#### New Release Creation:

> If a user has the permissions to create a release a 'New release' button will be present in the header. Clicking the button opens a form requiring a unique release name.

Reviewer: ```suggestion
If a user has the `plugin::content-releases.update` permissions, a 'New release' button will be visible in the header. Clicking the button opens a form requiring a name. The name of a pending release must be unique.
```
